organelle_morphology/organelle.py

```diff
@@ -142,7 +142,7 @@ def morphology_map(self):
         """Get the mesh data for this organelle"""
         comp_level = self._source._project.compression_level
 
-        morph_radius = 0.3
+        morph_radius = 0.0
```

Comment:
Can you maybe add a comment what the morph_radius is? It's otherwise hard to judge for me whether this was intentional or accidental change and what its impact is.